src/modules/github/components/AboutPRButton.tsx

```diff
@@ -0,0 +1,99 @@
+import axios from 'axios'
+import { type ChangeEvent, type FC, useState } from 'react'
+
+import { EPullRequestType, type TRepoHasCheck } from '@/types/github/root/index'
+import { PullReviewResponse } from '@/types/github/root/server'
+
+interface AboutPRButtonProps {
+  repoHasCheckArray: TRepoHasCheck[]
+}
+
+export const AboutPRButton: FC<AboutPRButtonProps> = ({
+  repoHasCheckArray
+}) => {
+  const [commentInput, setCommentInput] = useState<string>('')
+
+  const handleCommentChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setCommentInput(event.target.value)
+  }
+
+  const hasChecked = repoHasCheckArray.some((data) =>
+    data.pulls.some((pull) => pull.isChecked === true)
+  )
+
+  const reviewPullRequest = (
+    owner: string,
+    repo: string,
+    pullRequestNumber: number,
+    event: EPullRequestType,
+    body: string
+  ) => {
+    axios
+      .post<PullReviewResponse>(
+        `/api/github/repos/${owner}/${repo}/pulls/${pullRequestNumber}/reviews`,
+        {
+          event,
+          body
+        }
+      )
+      .then((response) => response.data)
+      .then((result) => {
+        console.log('result', result)
+      })
+      .catch(console.error)
+  }
+
+  const handleCommentClick = () => {
+    repoHasCheckArray.map((data) =>
+      data.pulls.map((pull) => {
+        if (pull.isChecked && pull.user.login) {
+          reviewPullRequest(
+            data.org,
+            data.repo,
+            pull.number,
+            EPullRequestType.COMMENT,
+            commentInput
+          )
+        }
+      })
+    )
```

Comment:
Nope, this is wrong. You need to learn the difference between `forEach` and `map`. You are not supposed to use `map` in here.

Reply:
okay, difference is map: new array return, foreach:nothing return.